picard/ui/__init__.py

```diff
@@ -75,10 +116,19 @@ def restore_geometry(self):
             self.restoreGeometry(geometry)
         elif self.defaultsize:
             self.resize(self.defaultsize)
+        splitters = config.persist[self.splitters_name()]
+        if splitters:
+            splitter_items = self._get_splitter_items()
+            for splitter in splitter_items:
+                name = self._get_name(splitter)
+                if name in splitters:
+                    splitter.restoreState(splitters[name])
```

Comment:
I would also remove unused entries (this can happen after a change to object name)

See https://github.com/rdswift/picard/pull/5/commits/0a129ab520beb8abea369db1bfa05e503721f352